scripts/run-unittests.sh

```diff
@@ -39,22 +39,6 @@ test-hostname-verification() {
     test/tlscommunicationtest.py teest 'tee*est' fail
 }
 
-test-client-system-bundle() {
-    local arch=$1 openssl_dir
-    openssl_dir=$(openssl version -d | awk '{ print $2 }' | tr -d '"')
-    SSL_CERT_DIR=$openssl_dir/certs \
-    SSL_CERT_FILE=$openssl_dir/cert.pem \
-    run-test $arch stage/$arch/build/test/tlstest \
-        github.com 443 github.com
-}
-
-test-client-file-bundle() {
-    local arch=$1
-    run-test $arch stage/$arch/build/test/tlstest \
```

Comment:
The `run-test` function is not used anywhere after removing these test functions. It should also be removed.

Reply:
Removed, thanks.